### ISSUE TITLE
How many categories do you have on the bdd dataset? And where is the ground truth of the bdd test set?

### ISSUE THREAD

**Issue by britney-f** [cardwing/Codes-for-Lane-Detection#92]

**Comment by cardwing** [comment]
The labels of the BDD100K testing set are not released. I just use the training and validation set since the labels are available. More details can be seen in our arxiv paper which will be published soon.
